atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/AbstractDegradedClusterTest.java

```diff
@@ -58,14 +58,25 @@ public abstract class AbstractDegradedClusterTest {
             new HashMap<>();
 
     private String schemaAtStart;
+    private final boolean withSchemaVersionRecording;
 
-    @Before
+    public AbstractDegradedClusterTest(boolean withSchemaVersionRecording) {
+        this.withSchemaVersionRecording = withSchemaVersionRecording;
+    }
+
+    public AbstractDegradedClusterTest() {
+        this(true);
+    }
```

Comment:
Schema version recording is not used by the `OneNodeDownAvailabilityTest`, `TwoNodesDownAvailabilityTest`, or `ThreeNodesDownAvailabilityTest`. Although the former two tests are okay with it, `ThreeNodesDownAvailabilityTest` was failing because `getUniqueReachableSchemaVersionOrThrow()` throws an exception when no node is available.